### PR TITLE
Cover state

### PR DESCRIPTION
**Pull request by donalddalton**

### CODE REVIEW

File: Accepted/asc042.md [hunk 1]
```diff
@@ -8,23 +8,22 @@ Donald Dalton <donald@transcriptic.com>
 We should allow creation of `Refs` with initial cover states.
 
 #### **Proposal**
-Allow `Refs` to optionally specify an initial `cover` state from the set of cover types compatible with the corresponding container.
+Allow `Refs` to optionally specify an initial `cover` type from the set of cover types compatible with the corresponding container.
```
Comment (yangchoo) [inline Accepted/asc042.md]: Given the possible broad interpretation of the word "cover", it'll be good to further clarify and make clear that this field refers to
- the cover-type which a container is initially covered with

And not:
1) a compatible cover-type which the container could be covered with
2) whether the container is covered or not (i.e. cover-state)

File: Accepted/asc042.md [hunk 2]
```diff
@@ -8,23 +8,22 @@ Donald Dalton <donald@transcriptic.com>
 We should allow creation of `Refs` with initial cover states.
```
Comment (yangchoo) [inline Accepted/asc042.md]: As mentioned below, cover states is somewhat ambiguous, let's make the reference clearer that its a specific cover-type that's on the ref itself

File: Accepted/asc042.md (hunk 1 above)
Comment (donalddalton) [reply Accepted/asc042.md]: Yeah this makes sense. My latest changes should make things less ambiguous.

File: Accepted/asc042.md (hunk 2 above)
Comment (donalddalton) [reply Accepted/asc042.md]: Agreed!

**Review by yangchoo — APPROVED** [review]
thanks for addressing the changes and making them clearer! 

Accepted :+1:

**Pull request merged**